nipype/interfaces/base.py

```diff
@@ -1170,7 +1176,7 @@ def aggregate_outputs(self, runtime=None, needed_outputs=None):
                                                       self.__class__.__name__))
                 try:
                     setattr(outputs, key, val)
-                    _ = getattr(outputs, key)
+                    getattr(outputs, key)
```

Comment:
this was a weird traits error with dynamic traits, where this wouldn't work without the `_` . if things pass perhaps this is no longer necessary. but let's hold this thought in our heads in case people start reporting weird errors.